exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -92,9 +103,9 @@ func addSample(tsMap map[string]*prompb.TimeSeries, sample *prompb.Sample, label
 // 		TYPE-label1-value1- ...  -labelN-valueN
 // the label slice should not contain duplicate label names; this method sorts the slice by label name before creating
 // the signature.
-func timeSeriesSignature(t otlp.MetricDescriptor_Type, labels *[]prompb.Label) string {
+func timeSeriesSignature(metric *otlp.Metric, labels *[]prompb.Label) string {
 	b := strings.Builder{}
-	b.WriteString(t.String())
+	b.WriteString(getTypeString(metric))
```

Comment:
Is it necessary to check for "" here? Is `metric` guaranteed to always have a valid type?